src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/ResidentJudgeFtpaDecisionConfirmation.java

```diff
@@ -53,32 +67,43 @@ public PostSubmitCallbackResponse handle(
             case "partiallyGranted":
                 postSubmitResponse.setConfirmationBody(
                     "#### What happens next\n\n"
-                    + "Both parties have been notified of the decision. The Upper Tribunal has also been notified, and will now proceed with the case.<br>"
+                        + "Both parties have been notified of the decision. The Upper Tribunal has also been notified, and will now proceed with the case.<br>"
                 );
                 break;
 
             case "refused":
             case "notAdmitted":
                 postSubmitResponse.setConfirmationBody(
                     "#### What happens next\n\n"
-                    + "Both parties have been notified that permission was refused. They'll also be able to access this information in the FTPA tab.<br>"
+                        + "Both parties have been notified that permission was refused. They'll also be able to access this information in the FTPA tab.<br>"
                 );
                 break;
 
             case "reheardRule32":
             case "reheardRule35":
                 postSubmitResponse.setConfirmationBody(
                     "#### What happens next\n\n"
-                    + "Both parties will be notified of the decision. A Caseworker will review any Tribunal instructions and then relist the case.<br>"
+                        + "Both parties will be notified of the decision. A Caseworker will review any Tribunal instructions and then relist the case.<br>"
                 );
                 break;
 
             case "remadeRule31":
             case "remadeRule32":
-                postSubmitResponse.setConfirmationBody(
-                    "#### What happens next\n\n"
-                    + "Both parties have been notified of the decision.<br>"
-                );
+                YesOrNo isDlrmSetAsideEnabled
```

Comment:
can this be change to be consistent with how Thomas does it - 
![image](https://github.com/hmcts/ia-case-api/assets/113999496/4339f883-ac28-449a-93eb-2e9e32d4d6e8)